src/main/java/org/takes/rq/RqWithoutHeader.java

```diff
@@ -52,18 +53,16 @@ public RqWithoutHeader(final Request req, final CharSequence name) {
         super(
             new Request() {
                 @Override
+                @SuppressWarnings("unchecked")
                 public Iterable<String> head() throws IOException {
-                    final Collection<String> head = new LinkedList<String>();
                     final String prefix = String.format(
                         "%s:", name.toString().toLowerCase(Locale.ENGLISH)
                     );
-                    for (final String header : req.head()) {
-                        if (!header.toLowerCase(Locale.ENGLISH)
-                            .startsWith(prefix)) {
-                            head.add(header);
-                        }
-                    }
-                    return head;
+                    return new Concat<String>(
+                        req.head(),
+                        Collections.EMPTY_LIST,
```

Comment:
@super132 I think it is better to use `Collections.emptyList()` method instead of static field